Symfony/CS/Fixer/Contrib/AlignDoubleArrowFixer.php

```diff
@@ -148,10 +140,12 @@ private function injectAlignmentPlaceholders(Tokens $tokens, $startAt = null, $e
             }
 
             if ($token->equals(',')) {
-                do {
+                for ($i = $index; $i < $endAt - 1; ++$i) {
```

Comment:
More precisely, `$tokens->isArray($i + 1) || ` is the cause. Removing it solves the issue.

@SpacePossum This bug is very annoying. Could you fix it quickly? If not, I suggest a revert of this part and works for your related issue later.

cc @keradus